docs/AccessControl.md

```diff
@@ -113,6 +113,246 @@ This specifies that the file is expected to be transmitted from the caller to
 the listener and its name is `results.csv`.
 
 
+### Rejection codes
+
+The listener callback handler is also able to decide about rejecting the
+incoming connection. In a normal situation, the rejection code is predefined
+as `SRT_REJ_RESOURCE`. The handler can, however, sets its own rejection
+code. There are two number spaces intended for this purpose (as the range
+below `SRT_REJC_PREDEFINED` is reserved for internal codes):
+
+* `SRT_REJC_PREDEFINED` and above: predefined errors. Errors from this range
+(that is, below `SRT_REJC_USERDEFINED`) have their definitions provided in
+the `access_control.h` public header file. The intention is that applications
+using these codes understood the situation described by these codes standard
+way.
+
+* `SRT_REJC_USERDEFINED` and above: to be freely defined by the application.
+Codes from this range can be only understood if both applications know the
+code definitions of the other, and these should be used only after making
+sure that both applications understood these codes.
+
+The intention for the predefined codes is to be consistent with the HTTP
+standard codes, therefore there are following sub-ranges used:
+
+* 0 - 99: Reserved for unique SRT-specific codes (unused by HTTP)
+* 100 - 399: Info, Success and Redirection in HTTP, unused in SRT
+* 400 - 599: Client and server errors in HTTP, adopted by SRT
+* 600 - 999: unused in SRT
+
+This code can be set by using the `srt_setrejectreason` function.
+
+The SRT-specific codes are:
+
+### SRT_REJX_FAlLBACK
```

Comment:
```suggestion
### SRT_REJX_FAlLBACK  <-- is this failback or fallback?
```

Reply:
Of course it should be FALLBACK. Damned kezboard. This has an extra key right of the left shift, so shift isn't always correctly pressed...